inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -49,18 +49,24 @@
           {% if record.get('report_numbers') %}
           <div id="record-report-numbers">{{ record_report_numbers(record) }}</div>
           {% endif %}
-        {% if record.get('dois') or record.get('arxiv_eprints') %}
-          <div id="doi-eprint-experiment">
-            {% if record.get('dois') %}
-              {{ record_doi(record) }}
-            {% endif %}
-            {% if record.get('arxiv_eprints') %}
-              {{ record_arxiv(record) }}
-            {% endif %}
-          </div>
+          {% if record.get('number_of_pages') %}
+            <div> Number of pages: {{ record.number_of_pages }} </div>
+          {% endif %}
+          {% if record.get('dois') or record.get('arxiv_eprints') %}
+            <div id="doi-eprint-experiment">
+              {% if record.get('dois') %}
+                {{ record_doi(record) }}
+              {% endif %}
+              {% if record.get('arxiv_eprints') %}
+                {{ record_arxiv(record) }}
+              {% endif %}
+            </div>
+          {% endif %}
+          {% if record.get('accelerator_experiments') %}
+            <div> Experiment: {{ record_experiment(record) }} </div>
           {% endif %}
-        {% if record.urls or record.external_system_numbers %}
-          <div id="external_links">{{ record_links(record) }}</div>
+          {% if record.urls or record.external_system_numbers %}
```

Comment:
According to your changes in https://github.com/inspirehep/inspire-next/pull/2743/files  `record.external_system_numbers` should be `record.external_system_identifiers`